test/tests/functional/pbs_two_mom_hooks_resources_used.py

```diff
@@ -99,7 +99,7 @@ def test_Rrecord(self):
         test = []
         test += ['#PBS -N NodeFailRequeueTest\n']
         test += ['echo Starting test at `date`\n']
-        test += ['sleep 5\n']
```

Comment:
Nothing in this test requires this job script.  It also doesn't wait for the job to finish, so using the default sleep of 100 is probably best.  I'd remove the script, and just leave the sleep time alone.